.circleci/config.yml

```diff
@@ -106,6 +106,20 @@ jobs:
           command: |
             cd telemetry-airflow
             bash bin/test-parse
+  docs:
+    docker: *docker
+    steps:
+    - checkout
+    - *build
+    - run:
+        name: Install dependencies
+        command: pip install mkdocs markdown-include
```

Comment:
We'll want to have a step here to validate doc examples. I'll do it in a separate PR since this is already getting bloated.